assets/js/_helpers.js

```diff
@@ -201,15 +202,16 @@ window.helpers = window.helpers || {
         if (localStorageIsUsable) {
             return {
                 get: function (key) {
-                    if (!localStorage[key]) return;
+                    let storageItem = localStorage.getItem(key)
+                    if (!storageItem) return;
                     try {
-                        return JSON.parse(decodeURIComponent(localStorage[key]));
+                        return JSON.parse(decodeURIComponent(storageItem));
                     } catch(e) {
                         // Erase non parsable value
                         helpers.storage.remove(key);
                     }
                 },
-                set: function (key, value) { localStorage[key] = encodeURIComponent(JSON.stringify(value)); },
+                set: function (key, value) { localStorage.setItem(key, encodeURIComponent(JSON.stringify(value))); },
```

Comment:
Small formatting proposition, for legibility:
```suggestion
                set: function (key, value) {
                  let encoded_value = encodeURIComponent(JSON.stringify(value));
                  localStorage.setItem(key, encoded_value);
                },
```